Packs/WorkdayITAdmin/Integrations/Workday_IT_Admin/Workday_IT_Admin.yml

```diff
@@ -0,0 +1,118 @@
+category: IT Services
+commonfields:
+  id: Workday IT Admin
+  version: -1
+configuration:
+- defaultvalue: ""
+  display: Username
+  name: credentials
+  required: false
+  type: 9
+- defaultvalue: ""
+  display: Report Type
+  name: report_type
+  options:
+  - Full-Report
+  - Delta_Report
+  required: false
+  type: 15
+- defaultvalue: ""
+  display: Workday Report URL
+  name: report_url
+  required: true
+  type: 0
+- additionalinfo: Import schedule to get events
+  defaultvalue: ""
+  display: Fetch Events Frequency In minutes
+  name: fetch_events_time_minutes
+  required: false
+  type: 0
+- additionalinfo: Maximum number of events to fetch at a time
+  defaultvalue: "50"
+  display: Fetch Limit (Recommended less than 200)
+  name: fetch_limit
+  required: false
+  type: 0
+- additionalinfo: Used to notify any errors via email
+  defaultvalue: ""
+  display: Email Notification Ids (Separated by comma)
+  name: email_notification_ids
+  required: false
+  type: 0
+- defaultvalue: ""
+  display: SMTP Server Host
+  name: smtp_server
+  required: false
+  type: 0
+- defaultvalue: ""
+  display: SMTP Server Port
+  name: smtp_port
+  required: false
+  type: 0
+- defaultvalue: ""
+  display: From Email
+  name: from_email
+  required: false
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- defaultvalue: ""
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- defaultvalue: ""
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Workday IAM Integration
+display: Workday IT Admin
+name: Workday IT Admin
+script:
+  commands:
+  - arguments:
+    - description: 'Report start date time in format "%Y-%m-%dT%H:%M:%SZ" Ex: 2020-01-20T11:48:34.336-08:00'
+      name: from_date
+      required: true
+    - description: 'Report end date time in format "%Y-%m-%dT%H:%M:%SZ". Ex: 2020-01-20T11:48:34.336-08:00'
+      name: to_date
+      required: true
+    description: Gets the events based on from and to date. Based on how the report
+      is filtered on Workday, it may be completion date or effective date
+    name: workday-get-delta-user-report
```

Comment:
@dantavori Since we're using report as a service, the report can vary depending on how it's setup in Workday. There is no standard output and it's pretty flexible. Even the attribute names can change. That's why there is no context defined.